README.md

```diff
@@ -43,7 +43,9 @@ actor {
 
 ### Building from Source
 
-- Clone [motoko](https://github.com/dfinity/motoko) and [vscode-motoko](https://github.com/dfinity/vscode-motoko/tree/viper) into the same parent directory
-- Switch to the `viper` branch in both repositories
-- In your terminal, run `cd vscode-motoko` and then `npm run package` (this will rebuild the compiler bindings)
+- Clone [vscode-motoko](https://github.com/dfinity/vscode-motoko/tree/viper) into the same parent directory
```

Comment:
```suggestion
- Clone [vscode-motoko](https://github.com/dfinity/vscode-motoko/tree/viper)
```

Reply:
So much simpler now! 

Maybe we could specify `git clone https://github.com/dfinity/vscode-motoko --recurse-submodules` to set everything up in one swoop.

Reply:
good point let me try this flow...

Reply:
Okay, committed directly to `viper`. It worked!